trunk/admin.widget.class.php

```diff
@@ -32,7 +32,7 @@ public function add_plugin_page()
 
     protected function render_message($msg, $msg_type = 'error')
     {
-        echo "<div class='$msg_type notice is-dismissible'><p>$msg</p></div>";
+        echo '<div class="' . $msg_type . ' notice is-dismissible"><p>$msg</p></div>';
```

Comment:
i think using [sprintf](http://php.net/manual/en/function.sprintf.php) is even better than this.